sdk/tables/Azure.Data.Tables/src/TableConstants.cs

```diff
@@ -55,7 +55,7 @@ internal static class Odata
         {
             internal const string OdataTypeString = "@odata.type";
             internal const string EdmBinary = "Edm.Binary";
-            internal const string EdmBoolean = "Emd.Boolean";
+            internal const string EdmBoolean = "Edm.Boolean";
```

Comment:
Is this used anywhere? If not, let's just remove it along with any of the other unused values here.

Reply:
It is not.
I'm going to remove them.